DemoUI/index.html

```diff
@@ -0,0 +1,257 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Backups app</title>
+    <meta name="description" content="A privileged app stub">
+
+    <!--
+    viewport allows you to control how mobile browsers will render your content.
+    width=device-width tells mobile browsers to render your content across the
+    full width of the screen, without being zoomed out (by default it would render
+    it at a desktop width, then shrink it to fit.)
+    Read more about it here:
+    https://developer.mozilla.org/Mozilla/Mobile/Viewport_meta_tag
+    -->
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0"/>
+
+    <link rel="stylesheet" href="css/app.css">
+    <link rel="stylesheet" href="css/gaia-fonts/style.css">
+    <link rel="stylesheet" href="css/font-awesome.min.css">
+
+    <link href="css/header.css" rel="stylesheet">
+    <link href="css/lists.css" rel="stylesheet">
+    <link href="css/buttons.css" rel="stylesheet">
+    <link href="css/switches.css" rel="stylesheet">
+    <link href="css/layout.css" rel="stylesheet">
+    <link href="css/gaia-icons-embedded.css" rel="stylesheet">
+
+	  <script type="text/javascript" src="FFOSBR.js" id="FFOSBR"></script>
+
+    <!--
+    Inline JavaScript code is not allowed for privileged and certified apps,
+    due to Content Security Policy restrictions.
+    You can read more about it here: https://developer.mozilla.org/Apps/CSP
+    Plus keeping your JavaScript separated from your HTML is always a good practice!
+
+    We're also using the 'defer' attribute. This allows us to tell the browser that
+    it should not wait for this file to load before continuing to load the rest of
+    resources in the page. Then, once everything has been loaded, it will parse and
+    execute the deferred files.
+    Read about defer: https://developer.mozilla.org/Web/HTML/Element/script#attr-defer
+    -->
+    <script type="text/javascript" src="js/app.js" defer></script>
+
+    <!--
+    The following two lines are for loading the localisations library
+    and the localisation data-so people can use the app in their
+    own language (as long as you provide translations).
+    -->
+    <link rel="prefetch" type="application/l10n" href="data/locales.ini" />
+    <script type="text/javascript" src="js/libs/l10n.js" defer></script>
+  </head>
+  
+  <body style='overflow: scroll;'>
+      <section role='region' class='vbox fit' style='position: absolute'>
```

Comment:
Indentation seems to be inconsistent throughout due to modifications in different editors. There's probably an automated tool we can use to correct this.